hugo/content/contact.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contact
+toc: true
+---
+
+# Contact
+
+You can contact us in several ways:
+
+## Community Channels
+
+* Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
```

Comment:
Prefixing/Introing with _Our_ here is inconsistent with the other bullet points.

```suggestion
* [Issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
```

Reply:
I wanted to clarify with that, that it belongs to mumble.
Maybe I would reformulate it, like:
`[Issue Tracker in Mumble's Github Repo]`

Reply:
I think it should be clear from the context that this is an issue tracker related to Mumble :thinking:

Reply:
> I think it should be clear from the context that this is an issue tracker related to Mumble :thinking:

That might be correct.

Reply:
Ok, I will just name it `Issue tracker on GitHub`.